### ISSUE TITLE
Unable to build Lumina desktop 1.5.0 on Linux

### ISSUE THREAD

**Issue by texstar** [lumina-desktop/lumina#682]
I'm trying to build Lumina on PCLinuxOS and I get this error. Any ideas how to get past it? lib64x11-devel 1.6.7 is installed. 

/usr/bin/ld: .build/obj/lthemeengineplatformtheme.o: in function `lthemeenginePlatformTheme::syncMouseCursorTheme(QString)':
lthemeengineplatformtheme.cpp:(.text+0x3fa4): **undefined reference to `XDefineCursor'**
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:112: liblthemeengine.so] Error 1

**Comment by texstar** [comment]
FYI: I was able to compile the desktop by commenting out this line in lthemeengineplatformtheme.cpp:
     // XDefineCursor(curs->dpy, wins[w]->winId(), curs->cursors[i]);

The of course breaks setting the cursor theme in theme settings. LOL

**Comment by mremski** [comment]
@texstar You may have just needed to install the libXcursor-devel package if there is one for your OS.  I had to do that on a Fedora system

**Comment by texstar** [comment]
Yes we do have a ibXcursor-devel package but it is the current version of 1.2.0 released 2019-03-11. I may try to get the fedora 1.1.15 version from 2017 to build and see what happens.

**Comment by Izaic** [comment]
Any progress on this issue? Does Lumina now compile?

**Comment by texstar** [comment]
Sure it compiles as long as I leave this commented out. 
// XDefineCursor(curs->dpy, wins[w]->winId(), curs->cursors[i]);

**Comment by q5sys** [comment]
Are you having any trouble compiling the 1.6.x versions of Lumina?

**Comment by texstar** [comment]
Not anymore.
